### PR TITLE
IC-0000 Added integration test for messaging

### PR DESCRIPTION
**Pull request by mahalrmoj**
Added initial integration test for the "Action Plan Submitted" scenario.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -78,3 +78,8 @@ services:
 features:
   crs:
     notify-action-plan-submitted: false
+
+interventions-ui:
```
Comment (tomsmyers) [inline src/main/resources/application.yml:82]: we now have the interventions API and this service hard coding these URLs. i wonder if a shared configmap would be more appropriate for sharing these across the whole namespace?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsdeliusinterventionseventlistener/model/crs/Contact.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.hmppsdeliusinterventionseventlistener.model.crs
```
[the remark this answers is not in the document]
Comment (tomsmyers) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsdeliusinterventionseventlistener/model/crs/Contact.kt:1]: same with the notification request class.

File: src/main/resources/application.yml (hunk 1 above)
Comment (mahalrmoj) [reply src/main/resources/application.yml:82]: I think that's a great idea - perhaps we could have a session on it within the team during our dev talk

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsdeliusinterventionseventlistener/model/crs/Contact.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (mahalrmoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsdeliusinterventionseventlistener/model/crs/Contact.kt:1]: I can move these classes to a community-api package

**Pull request merged**